10/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM buildpack-deps:jessie
```

Comment:
We should probably start using stretch instead of jessie for the default image:
https://wiki.debian.org/LTS